sympy/solvers/inequalities.py

```diff
@@ -393,6 +393,9 @@ def solve_univariate_inequality(expr, gen, relational=True, domain=S.Reals, cont
         A Relational type output is expected or not
     domain : Set
         The domain over which the equation is solved
+    cont_domain: bool
```

Comment:
Perhaps just `continuous` would be a better choice? ( I am +0.3 on a name change, so not a big issue. What do you think?)